Framework/SINQ/src/PoldiTruncateData.cpp

```diff
@@ -68,7 +68,7 @@ void PoldiTruncateData::setTimeBinWidthFromWorkspace(
         "Workspace does not contain any data. Aborting.");
   }
 
-  const MantidVec &xData = workspace->readX(0);
+  auto &xData = workspace->x(0);
```

Comment:
@LamarMoore While `auto &` is perfectly correct we did a quick poll on slack and an overwhelming consensus was that `const auto &` was better for readability as you don't have to check function definition. Most other code I have seen has used `const auto &`, can we be consistent here?